src/main/java/com/englishtown/vertx/solr/impl/DefaultSolrService.java

```diff
@@ -54,11 +54,13 @@ public void query(VertxSolrQuery query, QueryOptions options, Handler<AsyncResul
         }
 
         try {
-            QueryRequest request = new QueryRequest(query);
```

Comment:
Can you explain why these changes are necessary?  

SolrCloud should be able to work with the same path:
https://wiki.apache.org/solr/SolrCloud#Distributed_Requests

Or am I missing something?